packages/@aws-cdk/cdk/lib/cloudformation/stack.ts

```diff
@@ -33,14 +33,24 @@ export class Stack extends Construct {
    * @returns The Stack object (throws if the node is not part of a Stack-rooted tree)
    */
   public static find(scope: IConstruct): Stack {
+    const curr = Stack.tryFind(scope);
```

Comment:
That’s an unsafe breaking change... maybe we can’t safely
Make it without checking all call sites...

Reply:
Why is this a breaking change? `find()` used to throw and still throws, `tryFind()` does not throw.